apps/sms/js/information.js

```diff
@@ -202,11 +202,9 @@ var VIEWS = {
     render: function renderGroup() {
       var participants = Threads.get(this.id).participants;
       this.renderContactList(participants);
-      ThreadUI.setHeaderContent({
-        id: 'participant',
-        args: { n: participants.length }
-      });
-      ThreadUI.setHeaderAction('back');
+      navigator.mozL10n.setAttributes(
```

Comment:
Yes we can still keep it for participant header, but report part is not necessary.